GVFS/GVFS.FunctionalTests/Tools/GVFSHelpers.cs

```diff
@@ -12,6 +12,8 @@ namespace GVFS.FunctionalTests.Tools
 {
     public static class GVFSHelpers
     {
+        public const string ModifiedPathsNewLine = "\r\n";

```

Comment:
Ideally you can make this private as it's not really interesting to the test logic